module/executiondatasync/execution_data/execution_data.go

```diff
@@ -1,10 +1,46 @@
 package execution_data
 
 import (
+	"errors"
+
 	"github.com/onflow/flow-go/ledger"
 	"github.com/onflow/flow-go/model/flow"
 )
 
+// ExecutionDataDBMode controls which db type to use.
+type ExecutionDataDBMode int
+
+const (
+	// ExecutionDataDBModeBadger uses badger db
+	ExecutionDataDBModeBadger ExecutionDataDBMode = iota + 1
+
+	// ExecutionDataDBModePebble uses pebble db
+	// execution nodes
```

Comment:
what's this?

Reply:
[9bf6848](https://github.com/onflow/flow-go/pull/6180/commits/9bf6848a5f042b89a82e9feec9b6ce7c64af6e7d)